roles/hyperion/tasks/main.yml

```diff
@@ -1,119 +1,146 @@
-#     - deb https://deb.nodesource.com/node_13.x bionic main
-#     - deb-src https://deb.nodesource.com/node_13.x bionic main
-
-# - name: Install Hyperion-History-API dependencies
-#   apt: 
-#     name:
-# #      - default-jre
-# #      - apt-transport-https
-# #      - rabbitmq-server
-# #      - redis-server
-#       - nodejs
-#     state: latest
-#     update_cache: yes
-
-# - name: Install PM2
-#   npm:
-#     name: pm2@latest
-#     global: yes
-
-# - name: Start PM2
-#   command: pm2 startup
-#   args:
-#     creates: /etc/systemd/system/pm2-root.service
-# - name: Enables the rabbitmq_management plugin
-#   rabbitmq_plugin:
-#     names: rabbitmq_management
-#     state: enabled
-
-# # Ensure that the vhost /test exists.
-# - name: Create rabbitmq vhost
-#   rabbitmq_vhost:
-#     name: /hyperion
-#     state: present
-
-# # Add user to server and assign full access control on / vhost.
-# # The user doesn't have permission rules for other vhosts
-# - rabbitmq_user:
-#     user: '{{ rabbitmq_user }}'
-#     password: '{{ rabbitmq_password }}'
-#     tags: administrator
-#     permissions:
-#       - vhost: /hyperion
-#         configure_priv: .*
-#         read_priv: .*
-#         write_priv: .*
-#     state: present
-
-# - name: Configure Redis
-#   lineinfile:
-#     path: '{{ redis_conf_path }}'
-#     regexp: '^supervised'
-#     line: supervised systemd
-
-# - name: Enable redis service
-#   service:
-#     name: redis
-#     state: restarted
-#     enabled: yes
-
-- name: Template Hyperion Dependency Install
-  template:
-    src: install_deps.sh.j2
-    dest: '{{ working_dir }}/src/Hyperion-History-API/ns_install_deps.sh'
-    mode: u+wrx
-    owner: '{{ deploy_user }}'
-    group: '{{ deploy_group }}'
-    
-- name: Install Hyperion Dependencies
-  shell: '{{ working_dir }}/src/Hyperion-History-API/ns_install_deps.sh'
+- name: Configure Elasticsearch - enable xpack security
+  shell: 'echo "xpack.security.enabled: true" | sudo tee -a {{ es_conf_path }}'
   args:
     executable: /bin/bash
-    chdir: '{{ working_dir }}'
   become: yes
-  become_user: '{{ deploy_user }}'
+  become_user: "{{ deploy_user }}"
 
-- name: Template Elasticsearch/Kibana Install
-  template:
-    src: install_es.sh.j2
-    dest: '{{ working_dir }}/src/Hyperion-History-API/ns_install_es.sh'
-    mode: u+wrx
+- name: Configure Elasticsearch - jvm.options
+  shell: 'sudo sed -ie "s/## -Xms4g/-Xms{{ jvm_heap_size }}/; s/## -Xmx4g/-Xmx{{ jvm_heap_size }}/" {{ es_jvm_opts_path }}'
+  args:
+    executable: /bin/bash
+  become: yes
+  become_user: "{{ deploy_user }}"
+
+
+- name: Create elasticsearch service override directory
+  file:
+    path: '/etc/systemd/system/elasticsearch.service.d/'
+    state: directory
     owner: '{{ deploy_user }}'
     group: '{{ deploy_group }}'
-    
-- name: Install Elasticsearch/Kibana
-  shell: '{{ working_dir }}/src/Hyperion-History-API/ns_install_es.sh'
+
+- name: Configure Elasticsearch - systemd override.conf
```

Comment:
@LuqWright for example! you can start a review or leave a single comment, provide attachments, links, etc! you can even tag/notify people :D